### PR TITLE
Add delete team button

### PR DESCRIPTION
**Pull request by dotiurin**
Added delete button to TeamItem component and logic

## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LEAR-53)

## Code reviewers

- [ ] @ashatna
- [ ] @ChristinaPastushchak
- [ ] @dotiurin
- [ ] @iryn34
- [ ] @tarassito
- [ ] @VasilHamulyak
- [x] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [ ] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/team/urls.py [hunk 1]
```diff
@@ -9,4 +9,5 @@
     url(r'^(?P<team_id>\d+)/event/', include('event.urls', namespace='event')),
     url(r'^(?P<team_id>\d+)/comment/', include('comment.urls', namespace='comment')),
     url(r'^new/$', views.TeamView.as_view(), name='team_create'),
+    url(r'^(?P<team_id>\d+)/$', views.TeamView.as_view()),
```
Comment (VitalyiKhlivnyi) [inline eventually/team/urls.py]: unnecessary line

File: eventually/static/src/containers/teamList/TeamItem.js [hunk 2]
```diff
@@ -93,17 +125,70 @@ export default class TeamItem extends React.Component {
...
+                        onClick={this.handleToggle}
+                        style={styles.button}
+                    />
+                    <div style={styles.footer}></div>
+                </div>
+            );
+        } else if (this.state.stateOfTeamView==1) {
+            teamView = (
+                <div style={styles.team}>
+                    <p>Are you sure u want to delete team '{this.state.name}'?</p>
```
Comment (VitalyiKhlivnyi) [inline eventually/static/src/containers/teamList/TeamItem.js]: replace "u" by "you"

File: eventually/static/src/containers/teamList/teamService.js [hunk 3]
```diff
@@ -26,5 +26,11 @@ const teamServicePut = (id, name, description, image) => {
     return axios.put(appPath + id + '/', {name, description, image});
 };
 
+const teamServiceDelete = id => {
+    return axios.delete(appPath+id+'/')
```
Comment (VitalyiKhlivnyi) [inline eventually/static/src/containers/teamList/teamService.js]: add spaces before and after plus sign

Comment (dotiurin) [reply eventually/static/src/containers/teamList/teamService.js]: fixed

File: eventually/static/src/containers/teamList/TeamItem.js (hunk 2 above)
Comment (dotiurin) [reply eventually/static/src/containers/teamList/TeamItem.js]: done

File: eventually/team/urls.py (hunk 1 above)
Comment (dotiurin) [reply eventually/team/urls.py]: fixed

**Review by VitalyiKhlivnyi — APPROVED** [review]

**Review by VasilHamulyak — APPROVED** [review]

**Review by iryn34 — APPROVED** [review]

**Pull request closed without merging**